index.js

```diff
@@ -0,0 +1,42 @@
+//Benton Green
+module.exports = (function() { 
+
+  function FizzBuzz(input1, input2){
+    this.input1 = input1 || "Fizzy";
+    this.input2 = input2 || "Buzzer";
+  }
+    
+  FizzBuzz.prototype.input = function(startNum, endNum){
+    this.startNum = startNum;
+    this.endNum = endNum + 1; //added one to stopNum so that it would include ending number in list
+    this.finalOutput = writeFizzBuzz(startNum,endNum,this.input1,this.input2);
+  };
+
+  FizzBuzz.prototype.output = function(){
+    return this.finalOutput; 
+  };
+
+  this.writeFizzBuzz = function(startNum, endNum, input1, input2) {
```

Comment:
This whole method actually doesn't work, at least not how you'd expect it to (and when you attempt to call it above, you're not actually calling it). Rather than `this.writeFizzBuzz = function () { }`, it needs to be `function writeFizzBuzz() { }`.